### DISCUSSION TITLE
Can't add text into the field

### DISCUSSION THREAD

**Discussion by l-mtc** [awcodes/filament-tiptap-editor#220] [Q&A] [answered]
I'm not sure if I've done something wrong but I can't seem to actually add any content in: BuilderComponent\Block::make('Heading')
                                    ->schema([
                                        TiptapEditor::make('heading')
                                            ->extraInputAttributes(['style' => 'min-height: 12rem;'])
                                            ->profile('default')
                                            ->label('Header Content'),

**Comment by l-mtc** [comment]
I'm getting this error: Uncaught (in promise) DOMException: Failed to execute 'define' on 'CustomElementRegistry': the name "hex-color-picker" has already been used with this registry

**Comment by l-mtc** [comment]
Seemed to be ok temporarily when I cleared the cache but since then no luck

**Comment by awcodes** [comment, marked as answer]
This was fixed in v3.1.9.

**Reply by l-mtc** [reply]
Thanks!

**Reply by l-mtc** [reply]
FYI I don't think this is fully fixed, I have the following code same error:
```
Toggle::make('add_field')
                                                    ->label('Add input field')
                                                    ->onColor('success')
                                                    ->onIcon('heroicon-m-magnifying-glass')
                                                    ->live(),
                                                Section::make('Input Details')
                                                    ->columns(2)
                                                    ->hidden(fn(Get $get): bool => !$get('add_field'))
                                                    ->schema([
                                                        TiptapEditor::make('text')
                                                            ->hidden(fn(Get $get): bool => !$get('add_field'))
                                                            ->extraInputAttributes(['style' => 'min-height: 6rem;'])
                                                            ->profile('default')
                                                            ->label('Text'),
                                                        TextInput::make('placeholder')
                                                            ->hidden(fn(Get $get): bool => !$get('add_field')),
                                                        TiptapEditor::make('helper_text')
                                                            ->hidden(fn(Get $get): bool => !$get('add_field'))
                                                            ->extraInputAttributes(['style' => 'min-height: 6rem;'])
                                                            ->profile('default')
                                                            ->label('Helper text'),
                                                    ])
```
I also saw this error: [tiptap warn]: Duplicate extension names found: ['bubbleMenu']. This can lead to issues.
And if I use minimal I had an error Undefined property: stdClass::$color

**Reply by awcodes** [reply]
"[tiptap warn]: Duplicate extension names found: ['bubbleMenu']. This can lead to issues."

This is an issue in the Tiptap core and nothing something I can do anything about.


"Undefined property: stdClass::$color"

No clue where this is coming from. I'm not seeing any issues with the minimal profile.

![Screenshot 2023-11-29 at 3 01 51 PM](https://github.com/awcodes/filament-tiptap-editor/assets/3596800/8a5c7c36-c887-47f5-8458-47a13c91f46f)

**Reply by l-mtc** [reply]
It seems to have resolved itself today after a composer upgrade
